client/src/pages/Followers.js

```diff
@@ -0,0 +1,188 @@
+import { useState, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Grid,
+  makeStyles,
+  Paper,
+  Tab,
+  Tabs,
+  Typography,
+} from "@material-ui/core";
+import AccountCircleIcon from "@material-ui/icons/AccountCircle";
+import axios from "axios";
+
+const useStyles = makeStyles(() => ({
+  followButton: {
+    height: 40,
+    borderRadius: 28,
+    width: 120,
+    marginLeft: "auto",
+  },
+  pfpPlaceholder: {
+    fontSize: 50,
+    marginRight: 10,
+  },
+  //TODO: Needs better name
+  followersPageItem: {
+    width: 750,
+  },
+}));
+
+//TODO: Consider having one list of users and adding an `isFollowing` property. Probably less overhead that way.
+//TODO: Add some more comments
+
+//Toggles the state of a user to be following or suggested, and makes the appropriate API call.
+//NOTE: isFollowing refers to if the user being toggled is currently being followed by our user, not to if the user will be
+//toggled to following.
+const toggleUserGroup___ = (setFollowing, setSuggested) => (isFollowing) => {
```

Comment:
why the underscore suffix?

Reply:
it's a convention I've been using to keep track of the order of a function. For instance, calling a function that returns a function which does "some task" might look like this `const doSomeTask = doSomeTask_(args)`. Is there a better way to track this?